### PR TITLE
Add manifest-from command

### LINKED ISSUE #21: Add from-channel --manifest command
**Is your feature request related to a problem? Please describe.**
We need to extract EAP base manifest from the EAP base channel file in the prospero jobs in CI.

**Describe the solution you'd like**
I want to work on this. Maybe something as from-channel --manifest

**Describe alternatives you've considered**
Manual parsing of the channel file yaml.

I will work on this, this is just so I won't forget to update the job :).

### PR DESCRIPTION
**Pull request by xstefank**
Resolves #21

### CODE REVIEW

File: src/main/java/org/wildfly/prospero/extras/channel/merge/ChannelMergeCommand.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ Channel merge(Stream<Channel> channels, URL url, String name, String description
 
     }
 
-    private static List<Channel> read(Path path) {
+    public static List<Channel> read(Path path) {
```
Comment (xstefank) [inline src/main/java/org/wildfly/prospero/extras/channel/merge/ChannelMergeCommand.java:68]: @spyrkob I decided it's not worth it to create a separate util class just for this. But up to you.

File: src/main/java/org/wildfly/prospero/extras/manifest/from/ManifestFromCommand.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        final ChannelManifest channelManifest = manifestFrom(channelList);
+
+        System.out.println(ChannelManifestMapper.toYaml(channelManifest));
+
+        return ReturnCodes.SUCCESS;
+    }
+
+    public static ChannelManifest manifestFrom(List<Channel> channelPaths) {
+        return channelPaths.stream()
+            .map(channel -> ChannelManifestMapper.from(channel.getManifestCoordinate().getUrl()))
```
Comment (spyrkob) [inline src/main/java/org/wildfly/prospero/extras/manifest/from/ManifestFromCommand.java:41]: @xstefank that will only work if the channel uses URL based manifest. Is that intended or should this work with GA(V) manifests as well?

Comment (xstefank) [reply src/main/java/org/wildfly/prospero/extras/manifest/from/ManifestFromCommand.java]: no, I forgot about this. I added the functionality, but I can't find an easy quick way to test it. Should it be tested?

Comment (spyrkob) [reply src/main/java/org/wildfly/prospero/extras/manifest/from/ManifestFromCommand.java]: I need to add more testing for this project as we use it more, but for now I think we're good

**Review by spyrkob — APPROVED** [review]

**Pull request merged**